tensorboard/webapp/widgets/card_fob/card_fob_controller_component.ts

```diff
@@ -266,6 +269,9 @@ export class CardFobControllerComponent {
   }
 
   getDraggingFobStep(): number {
+    if (!this.timeSelection) {
+      throw new Error('TimeSelection is not defined');
```

Comment:
I have the impression that if there's any kind of error in Angular app, all functionality just broken. Can we do something soft here like return null or even handle this in isDraggingLower/Higher?
Actually, not in this PR, I already saw some conflict here.

Reply:
Yeah that's true, I was hoping adding a loud failure here would make it more detectable when the app is being worked on. It really shouldn't be possible to drag the fob  when the `timeSelection` is `undefined` so this isn't something I would expect to occur during normal use. Would you prefer I just use a non null assertion in this function?

Reply:
I would prefer throw error when there is significant mistake (data loading) that makes the web app broken. Dragging the fob does not feel like one.
I would prefer have assertion then.

Reply:
I'll update this to use a non null assertion. If that situation occurred, a null pointer exception would be thrown so the web app would be broken